aspnetcore/signalr/streaming.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Use streaming in ASP.NET Core SignalR
+author: rachelappel
+description: 
+manager: wpickett
+monikerRange: '>= aspnetcore-2.1'
+ms.author: rachelap
+ms.custom: mvc
+ms.date: 06/07/2018
+ms.prod: aspnet-core
+ms.technology: aspnet
+ms.topic: article
+uid: signalr/streaming
+---
+
+# Use streaming in ASP.NET Core SignalR
+
+By [Brennan Conroy](https://github.com/BrennanConroy)
+
+ASP.NET Core SignalR supports streaming return values of server methods. This is useful for scenarios where fragments of data will come in over time. When a return value is streamed to the client, each fragment is sent to the client as soon as it is available, rather than waiting for all the data to become available.
```

Comment:
One use case if for when "fragments come in over time" (like telemetry). The other main use case is for breaking up large downloads so they can be interleaved with other messages. If you try and download a 5gb file in one message, no other messages can be sent during that time period. 

also would change
> each fragment is sent as soon as it **becomes** available